.pre-commit-config.yaml

```
@@ -3,23 +3,32 @@ repos:
     rev: v4.4.0
```

```suggestion
    rev: v4.6.0
```
... probably doesn't hurt.